test/extended/deployments/deployments.go

```diff
@@ -512,13 +257,57 @@ var _ = g.Describe("deployments: parallel: deployment generation", func() {
 			g.By(fmt.Sprintf("checking the generation for %s: %s", resource, generation))
 			o.Expect(generation).To(o.ContainSubstring("3"))
 
-			g.By("verifying the second deployment is marked complete")
-			err = wait.Poll(100*time.Millisecond, 1*time.Minute, func() (bool, error) {
-				rc, err := oc.KubeREST().ReplicationControllers(oc.Namespace()).Get(dcName + "-" + version)
+			g.By("verifying that observedGeneration equals generation")
+			err = wait.Poll(1*time.Second, 1*time.Minute, func() (bool, error) {
+				dc, _, _, err := deploymentInfo(oc, name)
 				o.Expect(err).NotTo(o.HaveOccurred())
-				return deployutil.IsTerminatedDeployment(rc), nil
+				return deployutil.HasSynced(dc), nil
 			})
+		})
+	})
+
+	g.Describe("paused", func() {
+		g.It("should never run a new deployment", func() {
+			resource, name, err := createFixture(oc, pausedDeploymentFixture)
+			o.Expect(err).NotTo(o.HaveOccurred())
+
+			_, rcs, _, err := deploymentInfo(oc, name)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			if len(rcs) != 0 {
+				o.Expect(fmt.Errorf("expected no deployment, found %#v", rcs[0])).NotTo(o.HaveOccurred())
+			}
+
+			out, err := oc.Run("deploy").Args(resource, "--latest").Output()
+			o.Expect(err).To(o.HaveOccurred())
+			o.Expect(out).To(o.ContainSubstring("cannot deploy a paused deploymentconfig"))
+
+			out, err = oc.Run("deploy").Args(resource, "--cancel").Output()
+			o.Expect(err).To(o.HaveOccurred())
+			o.Expect(out).To(o.ContainSubstring("cannot cancel a paused deploymentconfig"))
+
+			out, err = oc.Run("deploy").Args(resource, "--retry").Output()
+			o.Expect(err).To(o.HaveOccurred())
+			o.Expect(out).To(o.ContainSubstring("cannot retry a paused deploymentconfig"))
+
+			out, err = oc.Run("deploy").Args(resource, "--enable-triggers").Output()
+			o.Expect(err).To(o.HaveOccurred())
+			o.Expect(out).To(o.ContainSubstring("cannot enable triggers for a paused deploymentconfig"))
```

Comment:
I thought this restriction was lifted (according to the latest code and https://github.com/openshift/origin/pull/9086#discussion_r65699348)